pages/mesosphere/dcos/services/elastic/3.2.0-7.9.3/configuration/index.md

```diff
@@ -0,0 +1,389 @@
+
+In addition to placing the tasks on different zones/racks, the zone/rack information will be included in the Elastic's node.attr.zone attrribute and cluster.routing.allocation.awareness.attributes is set to "zone". This enables Elastic to ensure data is replicated between zones/racks and not to two nodes in the same zone/rack.
+
+## Custom Elasticsearch YAML
+
+Many Elasticsearch options are exposed via the package configuration in `config.json`, but there may be times when you need to add something custom to the `elasticsearch.yml` file. For instance, if you have written a custom plugin that requires special configuration, you must specify this block of YAML for the {{ model.techName }} service to use.
+
+Add your custom YAML when installing or updating the {{ model.techName }} service. In the DC/OS UI, click **Configure**. In the left navigation bar, click `elasticsearch` and find the field for specifying a custom Elasticsearch YAML. You must base64 encode your block of YAML and enter this string into the field.
+
+You can do this base64 encoding as part of your automated workflow, or you can do it manually with an [online converter](https://www.base64encode.org).
+
+<p class="message--note"><strong>NOTE: </strong>You must only specify configuration options that are not already exposed in <code>config.json</code>.</p>
+
+## Using Volume Profiles
+
+Volume profiles are used to classify volumes. For example, users can group SSDs into a “fast” profile and group HDDs into a “slow” profile. 
+
+<p class="message--note"><strong>NOTE: </strong>Volume profiles are immutable and therefore cannot contain references to specific devices, nodes or other ephemeral identifiers.</p> 
+
+[DC/OS Storage Service (DSS)](https://docs.d2iq.com/mesosphere/dcos/services/storage/1.0.0/) is a service that manages volumes, volume profiles, volume providers, and storage devices in a DC/OS cluster.
+
+Once the DC/OS cluster is running and volume profiles are created, you can deploy Elasticsearch with the following configuration:
+
+```bash
+cat > elastic-options.json <<EOF
+{
+	"master_nodes": {
+		"volume_profile": "elastic",
+		"disk_type": "MOUNT"
+	},
+	"data_nodes": {
+		"volume_profile": "elastic",
+		"disk_type": "MOUNT"
+	},
+	"ingest_nodes": {
+		"volume_profile": "elastic",
+		"disk_type": "MOUNT"
+	},
+	"coordinator_nodes": {
+		"volume_profile": "elastic",
+		"disk_type": "MOUNT"
+	}
+}
+EOF
+```
+```bash
```

Comment:
I think there needs to be a line break between code snippets for linting?

```suggestion

```bash
```